azure-pipelines.yml

```diff
@@ -121,6 +121,8 @@ stages:
     condition: or(eq(variables.isScheduled, true), eq(variables.isReleasePrep, true), eq(variables.isRelease, true), eq(variables.isManual, true))
     pool:
       vmImage: 'ubuntu-latest'
+    variables:
+      default_python_version: '3.8'
```

Comment:
I'm happy with how this looks right now since docker tests are **supplementary** to the existing tests, rather than replacing them (yet). Or if it's not too big of a lift, would it be ok to include a `matrix` of python versions, like we have below?

Reply:
Done. This is in the stage definition and there was only 1 `job` that used it so I converted that to using a `matrix`.

Reply:
👍🏼